backEnd/routes/userCRUD.js

```diff
@@ -52,23 +52,23 @@ router.get('/me', async (req, res) => {
 router.post("/login", async (req, res) => {
   const { username, password } = req.body;
   if (!username || !password) {
-    return res.status(400).json({ error: "Username and password are required." });
+    res.json({ message: "Username and password are required." });
   }
 
   const user = await prisma.user.findUnique({
     where: { username }
   });
 
   if (!user) {
-    return res.status(401).json({ error: "Invalid username" });
+    res.json({ message: "Invalid Username" });
```

Comment:
why not have a 401 not authorized here? i think the status code should make the frontend easier to handle rather than just a straight message?